spec/factories/application.rb

```diff
@@ -31,13 +31,13 @@
         "student0_application_code_samples": FFaker::Lorem.paragraph,
         "student0_application_learning_history": FFaker::Lorem.paragraph,
         "student0_application_skills": FFaker::Lorem.paragraph,
-        "student0_application_code_background": FFaker::Lorem.paragraph,
+        "student0_application_language_learning_period": FFaker::Lorem.paragraph,
```

Comment:
Sucks, but good to know. Seems we're doing something wrong with either serializing the data from the `ApplicationDraft` or retrieving in from the DB later... From what I remember the `User::MONTHS_LEARNING` array is Strings only.